### PR TITLE
Use "table" package for tabular output and word wrap support

### PR DESCRIPTION
**Pull request by trevor-scheer**
Federation results in particular can be quite long, causing some
word wrap pandemonium. This package supports word wrap and a
provides a cleaner look overall for tabular output.

This PR does _not_ replace all usages of `heroku-cli-util`s `table` function.
For output that is currently untested or that I couldn't test manually, I didn't
make the switch.

Successful `service:check`
image

Failing `service:check`
image

`service:list`
image

Successful `service:push`
image

Failing `service:push`
image

TODO:

- [x] Update CHANGELOG.md\* with your change (include reference to issue & this PR)
- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass

\*Make sure changelog entries note which project(s) has been affected. See older entries for examples on what this looks like.

### CODE REVIEW

File: packages/apollo/src/commands/service/check.ts [hunk 1]
```diff
@@ -51,18 +50,6 @@ const formatChange = (change: Change) => {
   };
 };
 
-const reshapeGraphQLErrorToChange = (
```
Comment (trevor-scheer) [inline packages/apollo/src/commands/service/check.ts]: Unused function

File: packages/apollo/src/commands/service/check.ts [hunk 2]
```diff
@@ -680,13 +654,22 @@ export default class ServiceCheck extends ProjectCommand {
...
-          { key: "service", label: "Service" },
-          { key: "field", label: "Field" },
-          { key: "message", label: "Message" }
-        ]
-      });
+      this.log(
+        table(
+          [
+            ["Service", "Field", "Message"],
+            ...compositionErrors.map(error => Object.values(error))
```
Comment (trevor-scheer) [inline packages/apollo/src/commands/service/check.ts]:
```suggestion
            ...compositionErrors.map(Object.values)
```

File: packages/apollo/src/commands/service/list.ts [hunk 3]
```diff
@@ -57,33 +57,28 @@ function formatHumanReadable({
...
-        // The default `printLine` will output to the console; we want to capture the output so we can test
-        // it.
-        printLine: line => {
-          result += `\n${line}`;
-        }
-      }
+          .sort((s1, s2) =>
+            s1.name.toUpperCase() > s2.name.toUpperCase() ? 1 : -1
+          )
+          .map(entry => Object.values(entry))
```
Comment (trevor-scheer) [inline packages/apollo/src/commands/service/list.ts]:
```suggestion
          .map(Object.values)
```

**Comment by JakeDawkins** [comment]
`apollo client:push` updates:

- i made failures a table, but didn't put the filepath in the table, since wrapping of the filename makes the filename unclickable
- I removed the filename from successes, since that shouldn't matter

Screen Shot 2019-09-25 at 12 33 54 PM
Screen Shot 2019-09-25 at 12 34 07 PM

**Pull request merged**